Airports/EGLF/Ownership.txt

```diff
@@ -9,12 +9,116 @@ BORDER:LFTWR
 DEPAPT:EGLF
```

Comment:
Add the following at the very top of Ownership.txt:

```
SECTOR:Only LFGND 1:0:0
OWNER:LFA:LFT:LFG

SECTOR:Only LFGND 2:0:0
OWNER:LFA:LFT:TCNW:TCNE:TCN:TCSW:TCSE:TCS:TC:LES:LEN:LE:LC:LD:LS:LWN:LNC:LWC:LWS:LSC:L:LM:O

SECTOR:Only LFTWR 1:0:0
OWNER:LFA:LFT

SECTOR:Only LFTWR 2:0:0
OWNER:LFA:TCNW:TCNE:TCN:TCSW:TCSE:TCS:TC:LES:LEN:LE:LC:LD:LS:LWN:LNC:LWC:LWS:LSC:L:LM:O

SECTOR:Only LFAPP:0:0
OWNER:LFA
```

Reply:
What about just having the external border of the RMA light up? i.e. not all their additional airspace.

Reply:
Either:
![image](https://user-images.githubusercontent.com/12017284/93710972-676f9280-fb42-11ea-9fcd-2c9a34cd08cb.png)

Or:
![image](https://user-images.githubusercontent.com/12017284/93711021-a271c600-fb42-11ea-9147-7d218f4593d9.png)

Note that if we opt for the former, there is no _simple_ way to display an online ring to SE/NW/NE (like we do for all other LTMA APP positions) because in the case that TC SW and either SE/NW/NE are the same controller, both would display.

So it's a choice between:

1. Show only TC SW the outline of the RMA
2. Show all TC controllers the online ring for LF APP

Reply:
@cbyworth @KHardern any thoughts on this one? I'm personally leaning toward the online ring because the RMA looks a little messier / doesn't distinguish TC sector lines vs. APP positions as well. But keen to hear what you think and then let's just go for something so hopefully this can get merged in...

Reply:
As time goes by my feelings on the matter weaken 😅 I can see an argument either way. Ultimately I (a) don't personally use the display lines and (b) don't control TC/AC so happy to defer to whatever my C1 colleagues think best.

Reply:
I agree, this just needs to be merged and gotten over with. It's only Farnborough 😄